src/libraries/System.Collections.Immutable/src/System.Collections.Immutable.csproj

```diff
@@ -49,6 +49,7 @@ System.Collections.Immutable.ImmutableStack&lt;T&gt;</PackageDescription>
     <Compile Include="System\Collections\Immutable\ImmutableArray_1.Builder.DebuggerProxy.cs" />
     <Compile Include="System\Collections\Immutable\ImmutableArray_1.Enumerator.cs" />
     <Compile Include="System\Collections\Immutable\ImmutableArray_1.cs" />
+    <Compile Include="System\Collections\Immutable\ImmutableArray_1.netstandard21.cs" Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'netstandard2.1'))" />
```

Comment:
Likewise, there's not netstandard TFM, would probably be simpler to use netcoreapp

Reply:
Fixed.